src/web/server/document.tsx

```diff
@@ -20,15 +22,37 @@ interface RenderToStringResult {
 export const document = ({ data }: Props) => {
     const { CAPI, NAV, linkedData } = data;
     const title = `${CAPI.headline} | ${CAPI.sectionLabel} | The Guardian`;
+    const pathToDist = process.env.NODE_ENV === 'production' ? '.' : 'dist';
+    // This is the stats file generated by webpack loadable plugin
+    const statsFile = path.resolve(`${pathToDist}/loadable-stats.json`);
+
+    // We create an extractor from the statsFile
+    const extractor = new ChunkExtractor({
+        statsFile,
+        entrypoints: ['react'],
+    });
+
+    // Wrap your application using "collectChunks"
```

Comment:
Suspect this is a copy paste comment from elsewhere

Reply:
yep still WIP, my main focus will be trying to understand how everything fits together here, there will be a lot of changes